docs/topics/installing-web-console.adoc

```diff
@@ -120,7 +120,7 @@ $ mvn clean install
 The `windup-maven-plugin` jar is installed in your local Maven repository.
 endif::[]
 ifdef::plugin-guide-offline[]
-. Download the link:{ProductDownloadURL}{IDEPluginFilename}-{ProductVersion}.zip[IDE Plugin archive file].
+. Navigate to  the link:{DownloadPageURL}[*Migration Toolkit for Applications* site] and download the appropriate IDE plugin.
```

Comment:
Remove extra space.
Also, it should be styled "Migration Toolkit for Applications download site", no bold.